### DISCUSSION TITLE
Is Supabase Storage S3 compatible?

### DISCUSSION THREAD

**Discussion by ilyabo** [supabase/supabase#15756] [Questions]
Is it possible to access it using the JS client @aws-sdk/client-s3?

**Comment by GaryAustin1** [comment]
No.  Although it currently uses s3 there is no direct access.  Also a single s3 bucket is shared securely by several/many Supabase instances.

**Reply by ilyabo** [reply]
Thanks! What if I use a self-hosted version?

**Reply by GaryAustin1** [reply]
I believe so, but I've not used it.
https://supabase.com/docs/guides/self-hosting/storage/config
https://github.com/supabase/storage-api#documentation
Also search here in discussions.
